### ISSUE TITLE
Update "Manage users" page to reference the "user task access restrictions" settings for user groups

### ISSUE THREAD

**Issue by christian-konrad** [camunda/camunda-docs#3665]
Users reported they missed out on the docs for the config options for task access restrictions for user groups. 

As a result, we should update "Manage users" and "user groups" pages to reference the "user task access restrictions" settings for user groups.

- https://docs.camunda.io/docs/components/console/manage-organization/manage-users/
- https://docs.camunda.io/docs/components/concepts/access-control/user-groups/

**Comment by akeller** [comment]
@volodymyr-melnykc I don't think it makes sense for you to own this issue, but I'm a little confused why it's on the HTO board to begin with. Is it just because the HTO team reported it? 

Is this as simple as adding a link to [user task access restrictions](https://docs.camunda.io/docs/components/concepts/access-control/user-task-access-restrictions/)? If so, you can PR or we can assign a tech writer to review these pages and add the link. Is this a fair assessment, or am I missing some context?

**Comment by volodymyr-melnykc** [comment]
> [@volodymyr-melnykc](https://github.com/volodymyr-melnykc) I don't think it makes sense for you to own this issue, but I'm a little confused why it's on the HTO board to begin with. Is it just because the HTO team reported it?
> 
> Is this as simple as adding a link to [user task access restrictions](https://docs.camunda.io/docs/components/concepts/access-control/user-task-access-restrictions/)? If so, you can PR or we can assign a tech writer to review these pages and add the link. Is this a fair assessment, or am I missing some context?

@akeller It's on the HTO board because the HTO team reported it.

Yes, we can add a link to [user task access restrictions](https://docs.camunda.io/docs/components/concepts/access-control/user-task-access-restrictions/) to clarify it for the users. Let's assign a tech writer to review the docs for this.

**Comment by akeller** [comment]
@christinaausley this should be a quick win, can you pick it up in the next two weeks?
